test/c/fd_fdstat_get_refresh.c

```diff
@@ -0,0 +1,11 @@
+#include <unistd.h>
+#include <stdlib.h>
+#include <assert.h>
+
+int main(void)
+{
+    isatty(1);
+    assert(malloc(65*1024) != NULL);
```

Comment:
can you explicitly call `__builtin_wasm_memory_grow` here?

Reply:
Good idea. I've done that and checked it still fails without the fix.